src/CMakeLists.txt

```diff
@@ -523,10 +522,20 @@ list(SORT server_SRCS)
 # Avoid source_group on broken CMake version.
 # see issue #7074 #7075
 if (CMAKE_VERSION VERSION_GREATER 3.8.1)
+	source_group(TREE ${CMAKE_CURRENT_BINARY_DIR} PREFIX "ProtoBuf Source Files" FILES ${proto_targets})
 	source_group(TREE ${PROJECT_SOURCE_DIR} PREFIX "Source Files" FILES ${client_SRCS})
 	source_group(TREE ${PROJECT_SOURCE_DIR} PREFIX "Source Files" FILES ${server_SRCS})
 endif()
 
+set(client_SRCS
+	${client_SRCS}
+	${proto_targets}
+)
+
+if (CMAKE_VERSION VERSION_GREATER 3.8.1)
```

Comment:
this if statement seems like it is not doing anything?

Reply:
Good catch